### PR TITLE
Fix GCC-8 build

### PR DESCRIPTION
**Pull request by beckerhe**
Two small code-fixes to make the GCC-8 build work again.

Fixes: #1295

### CODE REVIEW

File: OrbitQt/TutorialOverlay.h [hunk 1]
```diff
@@ -10,6 +10,7 @@
 #include <QPoint>
 #include <memory>
 #include <optional>
+#include <functional>
```
Comment (dpallotti) [inline OrbitQt/TutorialOverlay.h:13]: clang-format I guess.

File: OrbitService/Utils.cpp [hunk 2]
```diff
@@ -176,17 +176,16 @@ static std::optional<pid_t> ProcEntryToPid(const std::filesystem::directory_entr
...
-    if (val.has_value()) {
-      vec.emplace_back(std::move(val.value()));
-    }
+  fs::directory_iterator proc{"/proc"};
+  std::vector<pid_t> pids;
 
-    return std::move(vec);
-  };
+  for (const auto& entry : proc) {
+    if (auto pid = ProcEntryToPid(entry); pid) {
```
Comment (dpallotti) [inline OrbitService/Utils.cpp:183]: As you know well by now, I liked the use of `has_value` in the original code! 😄

**Review by dpallotti — APPROVED** [review]

File: OrbitQt/TutorialOverlay.h (hunk 1 above)
Comment (beckerhe) [reply OrbitQt/TutorialOverlay.h]: Yessss! I should go back to primary school and learn the alphabet. Thanks, fixed!

File: OrbitService/Utils.cpp (hunk 2 above)
Comment (beckerhe) [reply OrbitService/Utils.cpp]: Sure. Fine with me. :+1: I added the `has_value`. Thanks!

**Pull request merged**